### ISSUE TITLE
The training is slow，how much time do you spend training/epoch?

### ISSUE THREAD

**Issue by xiaoxiong74** [sstzal/STAR-FC#6]
The training is slow，how much time do you spend training /epoch?

**Comment by sstzal** [comment]
Hi, the time for training a epoch is very long. However, the training will converge very fast before an epoch. Observe the change of the loss and you very find it.

**Comment by hx121071** [comment]
> Hi, the time for training a epoch is very long. However, the training will converge very fast before an epoch. Observe the change of the loss and you very find it.

I Observe the loss from 0.6 to 0.02 just training 0.1 epoch,  can I stop the training process? And when should I stop training process?

**Comment by sstzal** [comment]
> > Hi, the time for training a epoch is very long. However, the training will converge very fast before an epoch. Observe the change of the loss and you very find it.
> 
> I Observe the loss from 0.6 to 0.02 just training 0.1 epoch, can I stop the training process? And when should I stop training process?

You can test on the Intermediate save model. And I remember that when the training loss drops to 0.02, the model has converged.

**Comment by sstzal** [comment]
Perhaps 0.01 loss. Poor memory.

**Comment by xiaoxiong74** [comment]
> s and you very find it.

thanks your reply. I train STAR-FC and  GCN-V with my own data respectively， GCN-V  is better in three test sets.

**Comment by sstzal** [comment]
> > s and you very find it.
> 
> thanks your reply. I train STAR-FC and GCN-V with my own data respectively， GCN-V is better in three test sets.

Fair comparison？

**Comment by sstzal** [comment]
> > s and you very find it.
> 
> thanks your reply. I train STAR-FC and GCN-V with my own data respectively， GCN-V is better in three test sets.

Fair comparison？

**Comment by hx121071** [comment]
> Perhaps 0.01 loss. Poor memory.

Thank you.
